### PR TITLE
adds example using digitalocean spaces tutorial

### PR DESCRIPTION
**Pull request by joshbrooks**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/remote/modify.md [hunk 1]
```diff
@@ -245,17 +257,19 @@ $ dvc remote modify myremote endpointurl \
...
 ```
 
-S3 remotes can also be configured entirely via environment variables:
+Besides that, any settings that are available for AWS S3 and described above, are
+available for an S3 compatible storage if it supports them.
+
+For example, the
+[tutorial for DigitalOcean spaces](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
+creates a space (equivalent to an AWS bucket) called `example-name` in the `nyc3`
+region. Your remote settings would be set like this:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:266]:
```suggestion
Besides that, any settings that are available for Amazon S3 (see previous
section) may be available for S3 compatible storage. For example, let's setup a
DVC remote using the `example-name`
[DigitalOcean space](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
(equivalent to a bucket in AWS) in the `nyc3` region:
```

File: content/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -227,6 +227,18 @@ these settings, you could use the following options.
   > - [ACL Overview - Permissions](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)
   > - [Put Object ACL](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html)
 
+
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:230]:
```suggestion
```

File: content/docs/command-reference/remote/modify.md [hunk 3]
```diff
@@ -245,17 +256,17 @@ $ dvc remote modify myremote endpointurl \
...
+DVC remote using the `example-name`
+[DigitalOcean space](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key)
+(equivalent to a bucket in AWS) in the `nyc3` region:
 
 ```dvc
-$ export AWS_ACCESS_KEY_ID='<my-access-key>'
-$ export AWS_SECRET_ACCESS_KEY='<my-secret-key>'
-$ dvc remote add -d myremote s3://mybucket/my/path
+$ dvc remote add -d myremote s3://example-name/path/to/use
+$ dvc remote modify myremote https://nyc3.digitaloceanspaces.com
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:267]:
```suggestion
$ dvc remote modify myremote endpointurl \
                             https://nyc3.digitaloceanspaces.com
```

File: content/docs/command-reference/remote/modify.md [hunk 4]
```diff
@@ -227,6 +227,17 @@ these settings, you could use the following options.
   > - [ACL Overview - Permissions](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)
   > - [Put Object ACL](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html)
 
+S3 remotes can also be configured entirely via environment variables:
+
+```dvc
+$ export AWS_ACCESS_KEY_ID='<my-access-key>'
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:233]: I agree about moving this to the main S3 section.

**Review by jorgeorpinel — APPROVED** [review]
Thanks @joshbrooks ! I did some copy edits and a fix.

@shcheklein can you double check this, please?

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Oops, may need to check the formatting here... ⏳

**Pull request merged**